### DISCUSSION TITLE
parse error : input size too long (xxxx > 512)

### DISCUSSION THREAD

**Discussion by prologic** [grafana/grafana#47748] [General]
I'm seeing errors like:

```
parse error : input size too long (xxxx > 512)
```

in our Grafana dashboards that use Loki as a datasource.

It seems Loki queries however in Explore work -- at least at a very basic level.

What is this error, what does it mean? Is there a bug? Did we misconfigured something? Are we ingesting data that's too large for something somewhere?

**Comment by ZoeQ** [comment]
I have the same problem, do you find the solution?

**Reply by prologic** [reply]
Yes we did. We solve this by fixing our Loki labelling configuration. We had some labels coming through Loki's promtail and being ingested that were far too long. We fixed that and the problem was resolved.

**Reply by ognjen-it** [reply]
So, can you share with us some examples of fixing loki labelling config?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
